introduction.md

```diff
@@ -1 +1,5 @@
-My name is nitsirk and I'm learning to code.  My favorite Code School path is Ruby!
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?